### ISSUE TITLE
Dermal sinus tract

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7714]
**Preferred term label:**
Dermal sinus tract

**Synonyms**

**Definition (free text, please give PubMed ID)**
A dermal sinus tract is an abnormality present at birth over the dorsal midline where an abnormal epithelialized connection
from the skin tracks inwards towards the spine, most commonly seen in the lumbosacral region
PMID:31484633

The dermal sinus tracks in from the skin and may end in the soft tissues, epidural space, or intradurally, where it is commonly associated with (generally benign) intradural inclusion tumours (including dermoid, epidermoid, or teratoma), thickened fatty filum, tethered
cord, or diastematomyelia (split cord syndrome). Dermal sinus tracts are different from innocuous coccygeal dimples, which are located lower within the gluteal cleft and are not associated with cutaneous stigmata; completely “classic” dimples do not require further workup or follow-up. See Figure 1 of PMID:31484633.

**Parent term (use hpo.jax.org/app)**
Closed neural tube defect 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
600145 
PMID: 17409324  1/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
